examples/menubar/menubar-1/js/PopupMenuItemLinks.js

```diff
@@ -173,6 +173,10 @@ MenuItem.prototype.setExpanded = function (value) {
 MenuItem.prototype.handleClick = function (event) {
   this.menu.setFocusToController();
   this.menu.close(true);
+  if (this.popupMenu) {
```

Comment:
same q?

Reply:
If the selected menuitem opens a (sub) menu, then this.popupMenu will be true.
We want to suppress default link behavior for these guys.

So, basically the same answer as above, except that the menuitems here (in file PopupMenuItemLinks.js) are in a menu (as opposed to the ones in MenubarItemLinks.js, above, which are in the menubar).

Reply:
got it! And thanks for adding the comments.